frontend/packages/knative-plugin/src/catalog/event-source-data.ts

```diff
@@ -42,4 +43,10 @@ export const getEventSourceCatalogProviderData = (
       ),
       provider: 'Red Hat',
     },
+    [EventSourceCephModel.kind]: {
+      description: t(
+        'knative-plugin~Used to connect OpenShift managed applications to Ceph notifications.',
```

Comment:
we may need to check on this test with Ceph/design team, or if they would provide this with CRD then need to update API to fetch it.

Reply:
```suggestion
        'knative-plugin~Used to connect Ceph bucket notifications to OpenShift managed applications.',
```